### ISSUE TITLE
Aria Label Should Apply to All Tables When Sticky/Scroll is Set

### ISSUE THREAD

**Issue by mcook103** [ant-design/ant-design#44802]
### What problem does this feature solve?

When an and-design Table component has sticky or scroll properties set, the aria-label is only applied to the body table (not for the header table). For accessibility purposes (for those who are visually impaired), both the header table and the body table should have the aria-label set.

### What does the proposed API look like?

When sticky or scroll properties are set on a Table component, all "aria" properties (for example <Table aria-label="test" sticky ... /> should to be applied to both the header and body tables.

**Comment by LiangJavaDS** [comment]
> aria-label

can you provide minimal reproducible example and environment?

**Comment by LiangJavaDS** [comment]
> ### What problem does this feature solve?
> 这个功能解决了什么问题？
> When an and-design Table component has sticky or scroll properties set, the aria-label is only applied to the body table (not for the header table). 当 and-design Table 组件设置了粘性或滚动属性时，aria-label 仅应用于正文表（不适用于标题表）。For accessibility purposes (for those who are visually impaired), both the header table and the body table should have the aria-label set.出于可访问性目的（对于视力障碍者），标题表和正文表都应设置 aria-label。
> 
> ### What does the proposed API look like?
> 提议的 API 是什么样的？
> When sticky or scroll properties are set on a Table component, all "aria" properties (for example <Table aria-label="test" sticky ... /> should to be applied to both the header and body tables.当在 Table 组件上设置粘性或滚动属性时，所有“aria”属性（例如 应该应用于标题表和正文表。

minimal reproducible example

**Comment by afc163** [comment]
Reproduce: https://codesandbox.io/p/sandbox/ji-ben-yong-fa-antd-5-27-2-forked-3rtjl3?file=%2Fdemo.tsx%3A72%2C14
